bigtable-hbase-1.x-parent/bigtable-hbase-1.x-mapreduce/pom.xml

```diff
@@ -52,24 +83,12 @@ limitations under the License.
       </exclusions>
     </dependency>
 
+    <!-- Test Group -->
     <dependency>
-      <groupId>org.apache.hbase</groupId>
-      <artifactId>hbase-server</artifactId>
-      <version>${hbase1.version}</version>
-    </dependency>
-
-    <!-- CVE Group: force update transitive deps to exclude CVEs -->
```

Comment:
does this need a rebase?

Reply:
no, I realized that those deps are provided by hadoop so we cant force upgrade them since hadoop workers will just insert them into our classpath